pandas/core/generic.py

```diff
@@ -3858,6 +3859,11 @@ def to_csv(
 
         {storage_options}
 
+        preserve_complex : bool, default False
```

Comment:
As commented in the issue, you can use the `dtype` argument in `read_csv` to read complex values already. I'm negative on this approach.